src/chroot/chroot.rs

```diff
@@ -140,10 +140,7 @@ fn enter_chroot(root: &Path) {
     let root_str = root.display();
     std::env::set_current_dir(root).unwrap();
     let err = unsafe {
-        chroot(CString::new(".")
-            .unwrap()
-            .as_bytes_with_nul()
-            .as_ptr() as *const libc::c_char)
+        chroot(CString::new(".").unwrap().as_bytes_with_nul().as_ptr() as *const libc::c_char)
```

Comment:
That's a simple `cargo fmt` change.

Given that `CString` is used in about 28 places (11 files), I think it would be better to give it a comprehensive look and update it throughout the code base as a single PR instead of just this single change.